src/ast/mod.rs

```diff
@@ -695,6 +695,8 @@ pub enum Expr {
         // https://cloud.google.com/bigquery/docs/reference/standard-sql/format-elements#formatting_syntax
         format: Option<CastFormat>,
     },
+    /// `DEFAULT` value of a column e.g. INSERT INTO tbl (a, b) VALUES ('foo', DEFAULT)
+    Default,
```

Comment:
is default a reserved keyword in all dialects/sql-standard? wondering if its not this might break behavior for some folks

Reply:
You are right, I will leave this one to a future PR